IfcDiscreteAccessory Rail Import BIMQ 2021-07-21T07:41:33.5120880Z (#187)
bSI-InfraRoom/IFC-Specification: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_BalanceWeightTensioner/Documentation.md

> Properties of WeightTensioner used in railway. The property set can be used by the predefined type TENSIONINGEQUIPMENT of IfcDiscreteAccessory

Reviewer: ```suggestion
Properties of WeightTensioner used in railway. The property set can be used by the predefined type TENSIONINGEQUIPMENT of IfcDiscreteAccessory.
```

> Properties of WeightTensioner used in railway. The property set can be used by the predefined type TENSIONINGEQUIPMENT of IfcDiscreteAccessory.

Reviewer 2: ```suggestion
Properties of a weight tensioner. The property set can be used by the predefined type TENSIONINGEQUIPMENT of IfcDiscreteAccessory.
```
